pythnet-price-feeds/evm.md

```diff
@@ -4,11 +4,11 @@ description: Consume Pyth Network prices in EVM applications
 
 # Pyth on EVM
 
-On-chain EVM programs can use the [Solidity SDK](https://github.com/pyth-network/pyth-sdk-solidity) to read Pyth prices. The off-chain portion of the application can use [pyth-evm-js](https://github.com/pyth-network/pyth-js/tree/main/pyth-evm-js) to generate price update transactions. This repository's [Quickstart](https://github.com/pyth-network/pyth-js/tree/main/pyth-evm-js#quickstart) includes an example of both the on- and off-chain code necessary to integrate with Pyth.
+On-chain EVM programs can use the [Solidity SDK](https://github.com/pyth-network/pyth-sdk-solidity) to read Pyth prices. The off-chain portion of the application can use [pyth-evm-js](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/ethereum/sdk/js) to generate price update transactions. This repository's [Quickstart](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/ethereum/sdk/js#quickstart) includes an example of both the on- and off-chain code necessary to integrate with Pyth.
 
 ## EVM price pusher
 
-[Pyth EVM price pusher](https://github.com/pyth-network/pyth-js/tree/main/pyth-evm-price-pusher)
+[Pyth EVM price pusher](https://github.com/pyth-network/pyth-crosschain/tree/main/price_pusher)
```

Comment:
we should probably move this section (as @0xfirefist has generalized the pusher), but i'm leaving that for later.